index.bs

```diff
@@ -2468,6 +2520,61 @@ devices=] that are rarely used or do not include a [=platform authenticator=]; o
 backup [=public key credential|credentials=] in case another [=authenticator=] is lost.
 
 
+### Storage modality ### {#sctn-authenticator-storage-modality}
+
+An [=authenticator=] can store a [=credential private key=] in one of two ways:
+
+ 1. In persistent storage embedded in the [=authenticator=], [=client=] or [=client device=], e.g., in a secure element.
+
+    A [=credential private key=] stored in this way is a [=client-side-resident credential private key=].
+
+ 1. By encrypting the [=credential private key=] such that only this [=authenticator=] can decrypt it and letting the resulting
+    ciphertext be the [=credential ID=] for the [=public key credential source=]. The [=credential ID=] is stored by the [=[RP]=]
+    and returned to the [=authenticator=] via the {{PublicKeyCredentialRequestOptions/allowCredentials}} option of
+    {{CredentialsContainer/get()}}, which allows the [=authenticator=] to decrypt and use the [=credential private key=].
+
+    This enables the [=authenticator=] to have unlimited storage capacity for [=credential private keys=], since the encrypted
```

Comment:
I'm wondering if we ought to be using  [public key credential sources](https://w3c.github.io/webauthn/#public-key-credential-source) in places like this rather than credential private keys -- or perhaps we will fix this all up thru the entire spec if we address https://github.com/w3c/webauthn/issues/905#issuecomment-395856738 and the relevant portion of issue #358?

Reply:
Yeah, I think that needs some additional surgery to the spec at large. See also https://github.com/w3c/webauthn/pull/979#discussion_r201702295

Reply:
agreed.